.github/workflows/continuous-integration-workflow.yml

```diff
@@ -42,6 +42,8 @@ jobs:
         uses: docker://postgres
         with:
           args: psql -v ON_ERROR_STOP=1 --username postgres -h postgres -c "CREATE USER chainlink NOSUPERUSER CREATEDB;"
+      - name: Install Postgres for CLI tools
+        run: wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - && echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" |sudo tee  /etc/apt/sources.list.d/pgdg.list && sudo apt update && sudo apt install -y postgresql-client-13
```

Comment:
My initial gut reaction was to go with the older version, but I think this is probably more likely to be backwards compatible and work with the older versions. I did some searching and it look like cross version is not officially supported, but generally seems to work. We should probably converge on a single supported PG version long term, for now this looks good.

Reply:
Newer pg_dump with older pg is ok.